wp-content/themes/csisnuclearnetwork/template-parts/entry-header.php

```diff
@@ -7,178 +7,26 @@
  * @since 1.0.0
  */
 
-$object = get_queried_object();
 
 $is_archive = is_archive(); //Not search, 404 or Analysis
-$is_page = is_page();
-$has_thumbnail = has_post_thumbnail();
+// $is_page = is_page();
 $post_type = get_post_type(); //News, updates, projects, events, programs
-$is_404 = is_404(); //404
+// $is_404 = is_404(); //404
 $is_search = is_search(); //Search
 $is_home = is_home(); //Analysis
-$page_for_posts = get_option( 'page_for_posts' );
 $is_tag = is_tag(); //Tag
-$is_series = is_tax('series'); //Series
+$is_tax = is_tax(); //Series, Analysis subtypes
 $is_category = is_category(); //Category
-$is_author = is_author(); //Author
+// $is_author = is_author(); //Author
 $is_single = is_single();
 $post_parent_id = wp_get_post_parent_id(get_the_ID());
 
-if ( $is_search || $is_tag || $is_category ) {
-	$image_URL = get_field( 'header_image', 'option' );
-} elseif ( $is_home || $is_series) {
-	$image_URL = get_the_post_thumbnail_url( $page_for_posts );
-} else {
-	$image_URL = get_archive_thumbnail_src( 'nuclearnetwork-fullscreen' );
-}
+	if ( $is_tax || $is_category || $is_tag || $is_search || ( $post_type === 'programs' && $is_single && $post_parent_id ) || $post_type === 'news' || $is_home || ( in_array( $post_type, array( 'events', 'programs', 'projects', 'updates' ) ) && $is_archive ) ) {  
```

Comment:
```suggestion
if ( $is_home || $is_archive || $is_search || ( $post_type === 'programs' && $is_single && $post_parent_id ) || in_array( $post_type, array( 'news' ) ) ) {
```

`$is_archive` will be true on category, tag, and taxonomy pages. Since it looks like all archive pages have the dark blue header, I don't think you need to specify which post types need it.